create_reference_tsv.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import re
+inFile = sys.argv[1]
+outFile = sys.argv[2]
+
+sampleDict = {}
+outList = []
+
+with open(inFile, 'r') as f:
+	lines = f.readlines()
```

Comment:
According to PEP8 it is recommended to use 4 spaces instead tab as indentation level. Personally I also find it more readable that way.